### ISSUE TITLE
Label of the graph incorrect

### ISSUE THREAD

**Issue by habonlaci** [bw1129/PIDtoolbox#1]
When selecting a point with the datacursor on the noise spectrogram it says deg/s  instead of hz
![image](https://user-images.githubusercontent.com/4699494/52350083-1459d200-2a28-11e9-8520-227b853ed211.png)

**Comment by bw1129** [comment]
yes, I edited the data cursor for use with the main plots. work in progress. Probably need a separate datacursor function for spec plots. Work in progress. Feel free to add this.

**Comment by bw1129** [comment]
fixed
